OrbitGl/TracepointTrack.cpp

```diff
@@ -81,4 +115,35 @@ float TracepointTrack::GetHeight() const {
 
 bool TracepointTrack::HasTracepoints() const {
   return !GOrbitApp->GetCaptureData().GetTracepointsOfThread(thread_id_).empty();
+}
+
+void TracepointTrack::OnPick(int x, int y) {
+  Vec2& mouse_pos = mouse_pos_[0];
+  canvas_->ScreenToWorld(x, y, mouse_pos[0], mouse_pos[1]);
+  mouse_pos_[1] = mouse_pos_[0];
+  picked_ = true;
+}
+
+void TracepointTrack::OnRelease() { picked_ = false; }
+
+std::string TracepointTrack::GetSampleTooltip(PickingId id) const {
+  static const std::string unknown_return_text = "Linux tracepoint event information missing";
+
+  auto user_data = time_graph_->GetBatcher().GetUserData(id);
+  CHECK(user_data && user_data->custom_data_);
+
+  const auto* tracepoint_event_info =
+      static_cast<const orbit_client_protos::TracepointEventInfo*>(user_data->custom_data_);
+
+  uint64_t tracepoint_info_key = tracepoint_event_info->tracepoint_info_key();
+
+  TracepointInfo tracepoint_info =
+      GOrbitApp->GetCaptureData().GetTracepointInfo(tracepoint_info_key);
+
+  return absl::StrFormat(
+      "<b>Tracepoint activity</b><br/>"
```

Comment:
Nit: I would call this "Tracepoint event"

Reply:
Sure. Absolutely no problem. Thank you! Will update the code!

Reply:
Done!